tx_btree.go

```diff
@@ -120,6 +120,40 @@ func (tx *Tx) Get(bucket string, key []byte) (value []byte, err error) {
 	}
 }
 
+func (tx *Tx) GetLen(bucket string, key []byte) (int, error) {
```

Comment:
1. It seems that just use the tx.Get to get the value of this key 😁.
2. `ValueLen` maybe a better name for this API.